src/pricing.ts

```diff
@@ -476,6 +477,46 @@ const recomputeDetailTotalsFromCompositePrice = (
   }, details || initialPricingDetails);
 };
 
+const BLACK_LISTED_KEYS: Set<keyof Price> = new Set([
```

Comment:
Can we pick a different name please :) ~"BLACK LISTED"~

```suggestion
const ENTITY_FIELDS_EXCLUSION_LIST: Set<keyof Price> = new Set([
```

Reply:
Sure, sounds good. :)